resources/js/pages/invoices/create.tsx

```
@@ -0,0 +1,314 @@
+import { Head } from '@inertiajs/react';
+import { Eye, Plus, Send, Trash2 } from 'lucide-react';
+import { type FormEvent, useRef, useState } from 'react';
```

ESLint

[error] Prefer using a top-level type-only import instead of inline type specifiers.

(import/consistent-type-specifier-style)